src/util/languageUtils.js

```diff
@@ -40,7 +40,8 @@ export const getLanguage = (
 	defaultLang = LANG_DEFAULT
 ) => {
 	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
+	// TODO - better comment
```

Comment:
>defaultLang is selected when the accept-language header and/or cookie request a language other than the ones supported

Is `defaultLang` always `en-US`?  It looks like it can be overridden but i'm unclear about the circumstances in which it happens.